src/python/bezier/_plot_helpers.py

```diff
@@ -62,14 +62,15 @@ def add_plot_boundary(ax, padding=0.125):
     )
 
 
-def add_patch(ax, color, pts_per_edge, *edges):
+def add_patch(ax, color, pts_per_edge, *edges, alpha=None):
     """Add a polygonal surface patch to a plot.
 
     Args:
         ax (matplotlib.artist.Artist): A matplotlib axis.
         color (Tuple[float, float, float]): Color as RGB profile.
         pts_per_edge (int): Number of points to use in polygonal
             approximation of edge.
+        alpha (Optional[float]): Alpha value of patch centre, between 0 and 1 inclusive.
```

Comment:
```suggestion
        alpha (Optional[float]): Alpha value of patch center, between 0 and 1 inclusive.
```

American spelling? For consistency?

Reply:
Ah yep, thanks!

I'm an aussie, so that spelling was mostly muscle memory for me

Reply:
No worries!